app/views/evidence/show.html.erb

```diff
@@ -30,11 +30,11 @@
             <div class="note-text-inner">
               <h4 class="header-underline">
                 <span class="text-truncate" title="Evidence for this instance">Evidence for this instance</span>
+                <%= render partial: 'shared/liquid_loading' %>
                 <%= render partial: 'actions' %>
               </h4>
-
-              <div class="content-textile" data-behavior="content-textile">
-                <%= markup(@evidence.content, liquid: true) %>
+              <div class="content-textile" data-behavior="content-textile liquid-async" data-path="<%= preview_project_node_evidence_path(current_project, @evidence.node, @evidence) %>" data-content="<%= @evidence.content %>" >
```

Comment:
should we turn this into a `content_tag` call?

I'm hesitant about the `data-content="<%%>"` call without any escaping of the user-provided content.

Reply:
Used `content_tag` and added a partial